### PR TITLE
mongoose: add 'throw' to the strict option for schemas

### PR DESCRIPTION
**Pull request by jendrikw**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: http://mongoosejs.com/docs/guide.html#strict
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/mongoose/index.d.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Type definitions for Mongoose 5.0.1
 // Project: http://mongoosejs.com/
-// Definitions by: simonxca <https://github.com/simonxca>, horiuchi <https://github.com/horiuchi>, sindrenm <https://github.com/sindrenm>, lukasz-zak <https://github.com/lukasz-zak>, Alorel <https://github.com/Alorel>
+// Definitions by: simonxca <https://github.com/simonxca>, horiuchi <https://github.com/horiuchi>, sindrenm <https://github.com/sindrenm>, lukasz-zak <https://github.com/lukasz-zak>, Alorel <https://github.com/Alorel>, jendrikw <https://github.com/jendrikw>
```
Comment (simonxca) [inline types/mongoose/index.d.ts]: Remove this unless you want to help maintain these definitions. 🙂

**Review by simon-xiong-ca — APPROVED** [review]

**Comment by simonxca** [comment]
@jendrikw Thanks!

File: types/mongoose/index.d.ts (hunk 1 above)
Comment (jendrikw) [reply types/mongoose/index.d.ts]: I actually would like to help maintaining some defintitions, so I will leave it in.

Comment (simonxca) [reply types/mongoose/index.d.ts]: Okay great!

**Comment by DanielRosenwasser** [comment]
Thanks for contributing!

**Pull request merged**